python/gink/impl/websocket_connection.py

```diff
@@ -74,6 +82,10 @@ def receive(self) -> Iterable[SyncMessage]:
         self._ws.receive_data(data)
         for event in self._ws.events():
             if isinstance(event, Request):
+                if self.auth_token and encodeToken(self.auth_token) not in event.subprotocols:
```

Comment:
Don't want to rely on there being an exact match, because people might use different capitalization (e.g. "token <token>" or "Token <token>"), and might use a different number of spaces, so it's best to decode from hex, then split and compare.